src/components/adminResourceInput.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
this component looks unused, can we get rid of it?

Reply:
yes, this was a test component - done